src/pkg/packaging/dir.props

```diff
@@ -4,6 +4,9 @@
 
   <PropertyGroup>
     <PackagingRoot>$(ProjectDir)src/pkg/packaging/</PackagingRoot>
+
+    <!-- Package prebuilt binaries -->
+    <UsePrebuiltPortableBinaries>false</UsePrebuiltPortableBinaries>
```

Comment:
I would suggest renaming this variable to _UsingPrebuiltPortableBinariesForInstallers_